freeipa/src/main/java/com/sequenceiq/freeipa/service/freeipa/user/PasswordService.java

```diff
@@ -175,6 +176,21 @@ private void internalSetPasswords(String operationId, String accountId, String a
         }
     }
 
+    private Optional<UserManagementProto.WorkloadPasswordPolicy> getPasswordPolicyForUser(UserManagementProto.Account account, String userCrn) {
+        Crn crn = Crn.safeFromString(userCrn);
+        switch (crn.getResourceType()) {
```

Comment:
could you fix this switch-case indentation?

Reply:
> could you fix this switch-case indentation?

Fixed.